doc/_sphinx/scripts/versions.js

```diff
@@ -33,6 +33,14 @@ function convertVersionsToHtmlLinks(versionsList, currentVersion) {
   return out;
 }
 
+function maybeAddWarning(currentVersion) {
+  if (currentVersion !== 'main') {
```

Comment:
Wouldn't this make the warning appear even if you have the current version selected?  Should it be:

```
function maybeAddWarning(currentVersion,data) {
if(currentVersion !== 'main' || currentVersion !== data.split('\n')[0].trim()){
```
Obviously, you need to pass data where you call this function.

Reply:
I will leave main and latest w/o warning
And add warning to all old versions linking to latest (not main)
Makes sense?

Reply:
Sounds good!